pages/installation/os/centos.rst

```diff
@@ -85,6 +85,9 @@ The last step is to enable Graylog during the operating system's startup::
   $ sudo systemctl start graylog-server.service
 
 
+.. note:: We higly recommend that you proxy the Graylog Webinterface and API with NGINX/Apache like described in: :ref:`configuring_webif_nginx`.
```

Comment:
same as my comment in https://github.com/Graylog2/documentation/pull/125#issuecomment-231743893